src/Microsoft.PowerShell.CoreCLR.AssemblyLoadContext/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -400,6 +515,24 @@ public static void SetPowerShellAssemblyLoadContext([MarshalAs(UnmanagedType.LPW
             PowerShellAssemblyLoadContext.InitializeSingleton(basePaths);
         }
     }
+
+    /// <summary>
+    /// Test hooks for PowershellAssemblyLoadContext
+    /// </summary>
+    public static class PowerShellAssemblyLoadContextTestHooks
```

Comment:
We have [public static class InternalTestHooks](https://github.com/PowerShell/PowerShell/blob/578f9e5e8b86835e5c40be4a9306adf91d5899af/src/System.Management.Automation/engine/Utils.cs#L1588) for test hooks.